### PR TITLE
release 0.13.53

### PR DESCRIPTION
**Pull request by nickchapman-da**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,61 @@ Release notes
...
+
+- Ledger API Test Tool default tests modified. Use ``--list`` for the updated
+  list of default tests. Time service test dropped from the suite.
+
+[Sandbox]
+~~~~~~~~~
+- Static time mode is being deprecated in the future. A warning has been added
+  to notify users of this fact.
+
+- Scenarios are being deprecated in the future, in favor of `DAML Script
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]:
```suggestion
- Scenarios for ledger initialization are being deprecated in the future, in favor of `DAML Script
```

File: docs/source/support/release-notes.rst [hunk 2]
```diff
@@ -6,6 +6,61 @@ Release notes
...
+  <https://docs.daml.com/daml-script/>`_. A warning has been added to notify
+  users of this fact.
```
Comment (cocreature) [inline docs/source/support/release-notes.rst]:
```suggestion
  users of this fact. Scenarios can still be used for testing in DAML studio.
```

**Comment by nickchapman-da** [comment]
All manual tests have passed.

**Pull request merged**